src/pages/hegel/reference/mechanical-object/section-2.mdx

```diff
@@ -0,0 +1,24 @@
+	The first paragraph of “section 2” elaborates on the precise nature of the determinateness of the “mechanical object”. Hegel tackles the tension that has been present ever since we made explicit the essential determinateness of the “mechanical object” that is equally indeterminate. What does it mean for the “mechanical object” to be both essentially determinate and indeterminate? Hegel’s answer is that the “mechanical object” is indifferent [Gleichgültig] to its determinateness. As we said earlier, the “mechanical object” is able to be essentially determinate whilst being indeterminate because its moments of determinateness are immediately identical to each other. What does this tell us about these moments? It tells us that they are indifferent to each other – it is indifference that explains how a moment can be both a determinateness and indeterminate in that determinateness. As Hegel writes, the “mechanical object” is “indifferent to the determinations as individual, as determined in and for themselves, just as the latter are themselves indifferent to one another” (713). As such, since each determination is indifferent to every other determination, no determination can be comprehended from any other determination. The idea being that if a does not actively relate to b (does not posit b) then it would not make sense to try to understand b by understanding a. To use an empirical example, if you have two rocks, Ra and Rb, it would not make sense to seek to understand Rb by trying to understand Ra since what it is for Ra to be is for it to be indifferent to Rb: Ra is perfectly able to be itself without any explicit reference to Rb and any reference that there is to Rb is merely a formal one¸ i.e. merely in virtue of the fact that both rocks happen to be in the same place. This is what Hegel means when he concludes the first paragraph of Section 2 with a clarification of how the “mechanical object” can have both determinateness and be indeterminate:
```

Comment:
You must have 1 line between paragraphs in markdown in order to designate paragraphs.